sirepo/package_data/static/js/raydata.js

```diff
@@ -616,6 +616,61 @@ SIREPO.app.directive('pngImage', function(plotting) {
     };
 });
 
+SIREPO.app.directive('elementPicker', function() {
```

Comment:
I think this name could be improved to something more specific. Maybe columnPicker?

Reply:
07e13b7b7da132d5450c7714b0bcd7a04982308c